### ISSUE TITLE
osdc-bot

### ISSUE THREAD

**Issue by vnzongzna** [osdc/community-committee#12]
Few months back I made a simple [osdc-bot](https://github.com/vaibhavk/osdc-bots) which was meant to be a [monorepo](https://developer.atlassian.com/blog/2015/10/monorepos-in-git/) for other bot projects for OSDC. Currently it only contains source code of greeter bot for OSDC's [telegram group](https://t.me/jiitosdc) and would benefit the community if more people contribute to it.

What more can be done:
* Add the repository to osdc's group
* IRC bot

**Comment by ryzokuken** [comment]
+1 from me.

**Comment by ayushnagar123** [comment]
I guess we had an awesome gitter bot what about very similar to it.

**Comment by sidntrivedi** [comment]
@vaibhavk It will really be awesome. May I add the IRC bot I made in Ruby to the repo? For now, we can propose a project of IRC bot in node.js as suggested then by @ryzokuken.

**Comment by sidntrivedi** [comment]
> I guess we had an awesome gitter bot what about very similar to it.

Yes, here's the link to the repo - https://github.com/osdc/osdc-bot

**Comment by vnzongzna** [comment]
@ryzokuken is it possible to make the original osdc-bot repository a monorepo?

**Comment by vnzongzna** [comment]
> May I add the IRC bot I made in Ruby to the repo?

It would be great

**Comment by ryzokuken** [comment]
@vaibhavk everything is possible. Feel free to do that.
